mlflow/gateway/providers/utils.py

```diff
@@ -37,6 +40,7 @@ async def send_request(headers: Dict[str, str], base_url: str, path: str, payloa
                 response.raise_for_status()
             except aiohttp.ClientResponseError as e:
                 detail = js.get("error", {}).get("message", e.message) if "error" in js else js
+                _custom_token_allowance_exceeded_handling(detail, e.status)
```

Comment:
Can we catch `HTTPException` in `mlflow/gateway/providers/mosaicml.py` and reraise it as 422?

Reply:
Good idea. Moved it!